### PR TITLE
Test travis release

### PR DESCRIPTION
**Pull request by Velenir**
- Adds a file for release notes, which is supposed to work in place of `body`
- Lint fixes after a bunch of updates by @dependabot

### CODE REVIEW

File: src/Console.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const ConsoleFrame: React.FC = () => {
   const input = useRef<HTMLInputElement>(null)
 
   useEffect(() => {
-    const hookedConsole = Hook(window.console, (log: Message) => setLogs(logs => logs.concat(Decode(log))))
+    const hookedConsole = Hook(window.console, (log: Message) => setLogs((logs) => logs.concat(Decode(log))))
```
Comment (alfetopito) [inline src/Console.tsx:70]: What lint rules have changed? I couldn't find it in this PR

Comment (Velenir) [reply src/Console.tsx:70]: I think it was the bunch of updates coming from dependabot
Prettier changed, for one.

**Comment by Velenir** [comment]
Why the heck is travis not building?
@W3stside didn't you change repo setting recently to fix that?

**Comment by W3stside** [comment]
> Why the heck is travis not building?
> @W3stside didn't you change repo setting recently to fix that?

yes

**Pull request merged**